airflow/config_templates/config.yml

```diff
@@ -366,6 +366,17 @@
       type: string
       example: "path.to.CustomXCom"
       default: "airflow.models.xcom.BaseXCom"
+    - name: additional_execute_contextmanager
+      description: |
+        Custom user function that returns a context manager. Syntax is "package.method".
+        Context is entered when operator starts executing task. __enter__() will be called
+        before the operator's execute method, and __exit__() shortly after.
+        Function's signature should accept two positional parameters - task instance
+        and execution context
+      version_added: 2.0.0
+      type: string
+      example: ~
```

Comment:
```suggestion
      example: my.path. my_context_manager
```

Reply:
This will break any airflow run, it will try to access `my.path.my_context_manager` every time `execute` is called

Reply:
No it won't, that is just to show an example in https://airflow.apache.org/docs/stable/configurations-ref.html

It would break if you add that to `default:` key

Reply:
Sorry, I was wrong - example field instead of default field :)